templates/core/build.html

```diff
@@ -62,80 +62,96 @@ <h1>Build an Ubuntu IoT image</h1>
 </section>
 
 <div {% if not openid %}class="u-disable"{% endif %}>
-  <section class="p-strip is-shallow js-step-1">
+  <section class="p-strip js-step-1">
     <div class="row">
-      <div class="col-8">
-        <h2>1 What board are you using?</h2>
+      <div class="col-8 u-sv3">
+        <h2><span class="u-number-circle">1</span> What board are you using?</h2>
       </div>
     </div>
     <div class="row js-boards">
-      <button class="col-2 p-card js-selection">
-        <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/8542c958-Raspberry+Pi+2.png" alt="">
-        <hr class="u-sv1">
-        <p class="p-card__content">Raspberry Pi2</p>
-      </button>
-      <button class="col-2 p-card js-selection">
-        <img class="p-card__thumbnail" src="https://assets.ubuntu.com/v1/c4a007b6-Raspberry+Pi+3.png" alt="">
-        <hr class="u-sv1">
-        <p class="p-card__content">Raspberry Pi3</p>
+      <button class="col-3 p-card js-selection u-no-margin--right" style="width:100%;height:100%;">
+        <img src="https://assets.ubuntu.com/v1/8542c958-Raspberry+Pi+2.png" alt="" style="height:130px;" class="u-hide--small">
+        <h4 class="js-name u-no-margin--bottom">Raspberry Pi2</h4>
```

Comment:
Forgive the nitpick - these titles are descendants of a `h2`, so should be `h3`.

```suggestion
        <h3 class="p-heading--four js-name u-no-margin--bottom">Raspberry Pi2</h4>
```